WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -68,7 +68,7 @@ private static EstimateSmartFeeResponse SimulateRegTestFeeEstimation(int confirm
 		/// <summary>
 		/// Estimates fees for 1w, 3d, 1d, 12h, 6h, 3h, 1h, 30m, 20m.
 		/// </summary>
-		public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc, EstimateSmartFeeMode estimateMode = EstimateSmartFeeMode.Conservative, bool simulateIfRegTest = false)
+		public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc, CancellationToken cancel, EstimateSmartFeeMode estimateMode = EstimateSmartFeeMode.Conservative, bool simulateIfRegTest = false)
```

Comment:
`CancellationToken` is 99% of the cases the last parameter in the parameters list.

Reply:
You are right, i will fix it